### PR TITLE
[Woo POS] UI changes on product info dialog

### LINKED ISSUE #12509: [WOO POS] UI changes on Product info dialog
Screenshot 2024-09-04 at 9 53 43 AM

### PR DESCRIPTION
**Pull request by AnirudhBhat**
Closes: #12509

### Description

This PR has UI changes as per Joe's suggestion. Specifically, it includes following changes:
1. Adjusted modal dialog width to 75% of the total screen width. This change applies to both the Product Info dialog and the Exit POS dialog.
2. Updated line heights and spacing in the Product Info dialog. To achieve this, I split the existing Primary and Secondary messages into Primary, Secondary, and Tertiary messages.
3. Removed the close icon from the Product Info dialog, as the primary button now handles the close action.

Screenshot 2024-09-04 at 9 53 43 AM

### Testing information

1. Navigate to Product info dialog (More menu -> POS mode -> Click on Simple products banner learn more -> Product info dialog)
2. Ensure the Product info dialog modal is 75% of the total screen width (Also check Exit POS dialog as it uses the same wrapper dialog)
3. Ensure the product info dialog's text spacing and line height are as per design
4. Ensure you don't see close icon in Product info dialog.

### The tests that have been performed

Tested Product info dialog and Exit POS dialog both on Light and Dark mode.

### Images/gif

https://github.com/user-attachments/assets/910e4d57-cc47-451b-830e-f8fd225d1aba

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [x] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/234">20.3</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by backwardstruck** [comment]
Before

Screenshot_20240904_132241




After

Screenshot_20240904_132014

**Comment by backwardstruck** [comment]
```
Ensure the Product info dialog modal is 75% of the total screen width (Also check Exit POS dialog as it uses the same wrapper dialog)
Ensure the product info dialog's text spacing and line height are as per design
Ensure you don't see close icon in Product info dialog.
```
All looks good.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/component/WooPosDialogWrapper.kt [hunk 1]
```diff
@@ -25,32 +28,38 @@ fun WooPosDialogWrapper(
     onDismissRequest: () -> Unit,
     content: @Composable AnimatedVisibilityScope.() -> Unit
 ) {
-    Box(contentAlignment = Alignment.Center) {
```
Comment (backwardstruck) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/common/composeui/component/WooPosDialogWrapper.kt:28]: The bot comment `[28 Missing ⚠️]` is misleading as this code was primarily modified.

**Review by backwardstruck — APPROVED** [review]
Tested in light and dark mode.

**Pull request merged**